### DISCUSSION TITLE
Balance exceeds error in lesson 9

### DISCUSSION THREAD

**Discussion by Darshposhiya29** [Cyfrin/foundry-full-course-cu#446] [Q&A]
after running : forge script script/Interactions.s.sol:FundSubscription --rpc-url $SEPOLIA_RPC_URL --private-key $PRIVATE_KEY --broadcast

i am getting this error:
Traces:
  [5086245] → new FundSubscription@0x5b73C5498c1E3b4dbA84de0F1833c4a029d90519
    └─ ← 25180 bytes of code

  [2561186] FundSubscription::run() 
    ├─ [2486510] → new HelperConfig@0xC7f2Cf4845C6db0e1a1e91ED41Bcd0FcC1b0E141
    │   └─ ← 11643 bytes of code
    ├─ [963] HelperConfig::activeNetworkConfig() [staticcall]
    │   └─ ← 10000000000000000 [1e16], 30, 0x8103B0A8A00be2DDC778e6e7eaa21791Cd364625, 0x474e34a077df58807dbe9c96d3c009b23b3c6d0cce433e59bbf5b34f823bc56c, 3453, 500000 [5e5], 0x779877A7B0D9E8603169DdbD7836e478b4624789
    ├─ [0] console::9710a9d0(00000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000d7d000000000000000000000000000000000000000000000000000000000000001646756e64696e6720537562736372697074696f6e3a2000000000000000000000) [staticcall]
    │   └─ ← ()
    ├─ [0] console::log(Using vrfCoordinator: , 0x8103B0A8A00be2DDC778e6e7eaa21791Cd364625) [staticcall]
    │   └─ ← ()
    ├─ [0] console::9710a9d0(00000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000aa36a7000000000000000000000000000000000000000000000000000000000000000c4f6e20436861696e49643a200000000000000000000000000000000000000000) [staticcall]
    │   └─ ← ()
    ├─ [0] VM::startBroadcast() 
    │   └─ ← ()
    ├─ [3558] 0x779877A7B0D9E8603169DdbD7836e478b4624789::transferAndCall(0x8103B0A8A00be2DDC778e6e7eaa21791Cd364625, 100000000000000000000 [1e20], 0x0000000000000000000000000000000000000000000000000000000000000d7d) 
    │   └─ ← "ERC20: transfer amount exceeds balance"
    └─ ← "ERC20: transfer amount exceeds balance"

== Logs ==
  Funding Subscription:  3453
  Using vrfCoordinator:  0x8103B0A8A00be2DDC778e6e7eaa21791Cd364625
  On ChainId:  11155111
Error: 
ERC20: transfer amount exceeds balance

**Comment by AnouarBF** [comment]
Yeah I got that error too, and I fixed it! 
Make sure that you have imported link token into your metamask and fund it with link. Then go to the FundSubscription contract and add if you didn't specify how much links to fund```uint96 public constant  LINK = 3 ether```

**Reply by save-the-planet08** [reply]
You can just have some LINK from https://faucets.chain.link/
